tools/postgres/diff_schema.js

```diff
@@ -49,43 +51,43 @@ function dump(parameter) {
 function initTmpDB() {
   const tempDbName = generateRandomName();
   const postgresDirname = path.dirname(POSTGRES_URL);
-  const postgresUrl = `${postgresDirname}/${tempDbName}`;
-  const dbName = execSync(`${scriptdir}/db_name.sh`, { env: { POSTGRES_URL: postgresUrl } })
+  const postgresUrl = postgresDirname + "/" + tempDbName;
+  const dbName = execSync(scriptdir + "/db_name.sh", { env: { POSTGRES_URL: postgresUrl } })
     .toString()
     .trim(); // "trim" to remove the line break
   if (dbName !== tempDbName) {
     console.log("Wrong dbName");
     process.exit(1);
   }
-  const dbHost = execSync(`${scriptdir}/db_host.sh`, { env: { POSTGRES_URL: postgresUrl } })
+  const dbHost = execSync(scriptdir + "/db_host.sh", { env: { POSTGRES_URL: postgresUrl } })
     .toString()
     .trim();
-  return {
+  return [
     dbName,
     dbHost,
     postgresUrl,
-  };
+  ];
 }
 
 function loadDataIntoDB(parameter, dbHost, dbName) {
   const fileNames = glob.sync(parameter);
-  const concatenateFileNames = fileNames.map(name => `-f ${name}`).join(" ");
+  const concatenateFileNames = fileNames.map(name => "-f " + name).join(" ");
   execSync(
-    `psql -U postgres -h ${dbHost} --dbname="${dbName}" -v ON_ERROR_STOP=ON -q ${concatenateFileNames}`,
-    { env: { PGPASSWORD: "postgres" } },
+    "psql -U postgres -h " + dbHost + " --dbname='" + dbName + "' -v ON_ERROR_STOP=ON -q " + concatenateFileNames,
+    { env: { PGPASSWORD: "postgres" } }
   );
 }
 
 function dumpToFolder(postgresUrl) {
-  const tmpDir = tmp.dirSync();
+  const tmpDir = execSync("mktemp -d").toString().trim();
   try {
-    execSync(`${scriptdir}/dump_schema.sh ${tmpDir.name}`, { env: { POSTGRES_URL: postgresUrl } });
+    execSync(scriptdir + "/dump_schema.sh " + tmpDir, { env: { POSTGRES_URL: postgresUrl } });
   } catch (err) {
-    console.log(`CLEANUP: remove ${tmpDir.name}`);
-    rimraf.sync(tmpDir.name);
+    console.log("CLEANUP AAA: remove " + tmpDir);
```

Comment:
Why "AAA"?

Reply:
sorry, that's just from my debugging, I'll remove it :D